-------------------------------------------------------------------------------
Test set: maybank.insurance.dao.InsurancedaoApplicationTests
-------------------------------------------------------------------------------
Tests run: 3, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.024 s <<< FAILURE! - in maybank.insurance.dao.InsurancedaoApplicationTests
callAllInsuranceAvailable_failure  Time elapsed: 0.009 s  <<< ERROR!
maybank.insurance.dao.exceptions.InsuranceAvailableException: No data found
	at maybank.insurance.dao.InsurancedaoApplicationTests.callAllInsuranceAvailable_failure(InsurancedaoApplicationTests.java:55)

callAllInsuranceAvailable_Success  Time elapsed: 0 s  <<< ERROR!
maybank.insurance.dao.exceptions.InsuranceAvailableException: No data found
	at maybank.insurance.dao.InsurancedaoApplicationTests.callAllInsuranceAvailable_Success(InsurancedaoApplicationTests.java:40)

